packages/story-editor/src/app/font/actions/useLoadFontFiles.js

```diff
@@ -37,7 +37,10 @@ function useLoadFontFiles() {
     const handle = cleanForSlug(family);
     const elementId = `web-stories-${handle}-font-css`;
 
-    const hasFontLink = () => document.getElementById(elementId);
+    const hasFontLink = () =>
```

Comment:
Is this change needed? Since this is within a callback and never called directly during render as far as I can see